torchgeo/datasets/oscd.py

```diff
@@ -130,12 +158,15 @@ def __getitem__(self, index: int) -> dict[str, Tensor]:
         image2 = self._load_image(files["images2"])
         mask = self._load_target(str(files["mask"]))
 
-        image = torch.cat([image1, image2])
-        sample = {"image": image, "mask": mask}
-
         if self.transforms is not None:
```

Comment:
This seems more complicated than it needs to be. Can we not get `self.transforms` to work on the new sample?